player/command.c

```diff
@@ -5489,29 +5501,78 @@ static void cmd_expand_path(void *p)
     };
 }
 
+static struct load_action get_load_action(struct MPContext *mpctx, int action_flag)
+{
+    switch (action_flag) {
+    case 0: { // replace
+        return (struct load_action){LOAD_TYPE_REPLACE, .play = true};
+    }
+    case 1: { // append
+        return (struct load_action){LOAD_TYPE_APPEND, .play = false};
+    }
```

Comment:
```suggestion
    case 0: // replace
        return (struct load_action){LOAD_TYPE_REPLACE, .play = true};
    case 1: // append
        return (struct load_action){LOAD_TYPE_APPEND, .play = false};
```

Could nuke all the extra parenthesis here. Ditto for the rest of it. More concise and easier to read imo.

Reply:
Gotcha, will do.